runtime/src/main/java/io/quarkiverse/web/bundler/runtime/devmode/ChangeEventHandler.java

```diff
@@ -119,6 +126,11 @@ private void onChange(Set<String> srcChanges) {
 
     }
 
+    private static void writeField(StringBuilder sb, String field, String value) {
+        // We shouldn't have any new lines in values for this service
+        sb.append(field).append(": ").append(value.replaceAll("\\v", "")).append(NL);
```

Comment:
That's not how it works for SSE: 

> The data field for the message. When the EventSource receives multiple consecutive lines that begin with data:, [it concatenates them](https://html.spec.whatwg.org/multipage/#dispatchMessage), inserting a newline character between each one. Trailing newlines are removed

So, if you have `\n` in your data, say `foo\nbar` you should encode it to:

```
data: foo
data: bar
```

See https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events and https://html.spec.whatwg.org/multipage/server-sent-events.html

Reply:
@FroMage in this case, I control what is sent and there should be no new line so this was easier.

Reply:
This fix is just to make sure that the json is on one line

Reply:
Still sounds like a bug waiting to happen, it doesn't cost much to implement it properly ;)

Reply:
I don't see the point of implementing something fancy when it's not needed. Worst case scenario it will remove new lines from json content.